src/app/containers/ConsentBanner/index.jsx

```diff
@@ -0,0 +1,102 @@
+import React, { Fragment } from 'react';
+import ConsentBanner from '../../components/ConsentBanner';
+import { PlatformContextConsumer } from '../../contexts/PlatformContext';
+import { ServiceContextConsumer } from '../../contexts/ServiceContext';
+
+const parentId = 'consent';
+const promptId = 'consent-prompt';
+
+const ampOnTapHandler = type => ({ on: `tap:${parentId}.${type}` });
+
+const ampGeoData = {
```

Comment:
Any chance this can be extracted somewhere? Perhaps we could have an issue for it?

Reply:
https://github.com/bbc/psammead/issues/298